### PR TITLE
Comfy frontend

### PR DESCRIPTION
**Pull request by ddevault**

### CODE REVIEW

File: _drafts/2016-06-03-Building-a-comfy-frontend-stack.md [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+
+In the JavaScript world today there are tons of shiny new projects out there and
+it's overwhelming to figure out how to build a large product from the ground up.
+Before starting on [the new Linode Manager](https://github.com/Linode/manager),
+we had a long period of internal prototyping and developer discussions to find
+the ideal frontend stack for the project. Three main prototypes emerged, based
+on [React.js](http://facebook.github.io/react/),
+[Angular](https://angularjs.org/), and [Mithril](http://mithril.js.org/). Each
+of these platforms ships a different broader design philosophy. React.js brings
+a lot of functional programming concepts to the table. Angular is a declarative
```
Comment (eatonphil) [inline _drafts/2016-06-03-Building-a-comfy-frontend-stack.md]: React.js brings functional programming or redux does?

File: _drafts/2016-06-03-Building-a-comfy-frontend-stack.md [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+
+Angular also offers some innovative ways of managing your data with the
+observables mechanic. They basically work like Unix pipes and let you chain
+operations together and subscribe to changes anywhere in the pipeline, then
+respond to those changes in your application. Angular is also well tested and
+the community has many examples, though some may be outdated (which is a problem
+for all three of the options we're looking at today).
+
+There are some disadvantages to Angular. Though Google has promised improvements
+to this, Angular 2 is pretty slow. There's lots of debugging things going on and
```
Comment (eatonphil) [inline _drafts/2016-06-03-Building-a-comfy-frontend-stack.md]: s/There's/there are/

File: _drafts/2016-06-03-Building-a-comfy-frontend-stack.md (hunk 1 above)
Comment (ddevault) [inline _drafts/2016-06-03-Building-a-comfy-frontend-stack.md]: Both do.

File: _posts/2016-05-26-Getting-started-with-the-new-manager.md [hunk 3]
```diff
@@ -23,6 +23,10 @@ is a box. Click Manage API clients. Then fill out the application name
 (Linode Manager) and callback URL (http://localhost:3000/oauth/callback). Then
 hit Register. Make sure to save the client id and secret to a file.
 
+Note: if you pick a callback url that is not on localhost:3000, you will need to
```
Comment (eatonphil) [inline _posts/2016-05-26-Getting-started-with-the-new-manager.md:26]: What's with this?

File: _drafts/2016-06-03-Building-a-comfy-frontend-stack.md [hunk 4]
```diff
@@ -0,0 +1,212 @@
...
+
+I also asked Abe about the MVC pattern in general, and he still feels like it's
+the right call in some situations. Abe worries that once your application gets
+big enough and cross dependencies start to pile up, MVC falls apart. [Facebook
+feels the same way](http://facebook.github.io/flux/) (though I think the
+prevailing thought is that Flux isn't quite right, either).
+
+## React.js & ES6
+
+React.js is Facebook's offering for the new age of JavaScript, and Redux is a
```
Comment (eatonphil) [inline _drafts/2016-06-03-Building-a-comfy-frontend-stack.md:109]: could insert brief historical reference to older technologies React was based on ([elm](http://elm-lang.org/), [clojurescript](https://github.com/clojure/clojurescript), and [js_of_ocaml](http://ocsigen.org/js_of_ocaml/)), but it's your call.

Comment (ddevault) [inline _drafts/2016-06-03-Building-a-comfy-frontend-stack.md:109]: I intend to dig deeper into React and the rest of our stack in future blog posts. May cover this there.

Comment (eatonphil) [inline _drafts/2016-06-03-Building-a-comfy-frontend-stack.md:109]: it's not necessarily that React/Redox are "based" on them - these technologies just existed before and provided similar ways of organization

**Pull request closed without merging**